Resources/Private/Data/MetadataDefaults.php

```diff
@@ -80,6 +80,25 @@
         'is_listed' => 1,
         'index_autocomplete' => 0,
     ],
+    'date' => [
+        'format' => [
+            [
+                'format_root' => 'mods',
+                'xpath' => './mods:originInfo[@eventType="publication"]/mods:dateIssued[@encoding="iso8601"][@keyDate="yes"]',
```

Comment:
We should at least drop the `[@eventType="publication"]` condition and also allow the `w3cdtf` encoding in order to be applicable to a broader range of documents. I think, the most important indicator should be `[@keyDate="yes"]`. But it's still limited to `dateIssued`, so maybe we can even include `dateCreated` as a fallback? What do you think?

```suggestion
                'xpath' => './mods:originInfo/mods:dateIssued[@encoding="iso8601" or @encoding="w3cdtf"][@keyDate="yes"]',
```

Reply:
I agree with removing the `[@eventType]` attribute and with adding the `w3cdtf` encoding. We could include `dateCreated` as a fallback...but the best i can think of is this XPATH 1.0 expression, that might return a set of two dates in some cases:

`./mods:originInfo/*[local-name()="dateIssued" or local-name()="dateCreated"][@encoding="iso8601" or @encoding="w3cdtf"][@keyDate="yes"]`

Reply:
went shorter on it, to take any `keyDate` with the specified encoding:
`./mods:originInfo/*[@encoding="iso8601" or @encoding="w3cdtf"][@keyDate="yes"]`